pkg/web/src/hooks.client.ts

```diff
@@ -0,0 +1,40 @@
+import {getWebInstrumentations, initializeFaro} from '@grafana/faro-web-sdk';
+import {TracingInstrumentation} from '@grafana/faro-web-tracing';
+import {PUBLIC_BACKEND_ENDPOINT} from '$env/static/public'
+
+function setupFaro() {
+    const res = fetch(`${PUBLIC_BACKEND_ENDPOINT}api/config`).
+    then(data => data.json()).
+    then(config => {
+        const url = config.faro_url;
+        if (url === undefined) {
+            console.warn("Grafana faro is not configured.")
+            return
+        }
+
+        console.log(`Initializing Grafana Faro to ${url}`)
+        initializeFaro({
+            url: url,
```

Comment:
```suggestion
            url,
```

Reply:
I am once again blind to JS magicks 🧙🏻‍♀️